_posts/2021-09-06-connector-table-sql-api-part2.md

```diff
@@ -0,0 +1,519 @@
+---
+layout: post
+title:  "Implementing a custom source connector for Table API and SQL - Part Two "
+date: 2021-09-06T00:00:00.000Z
+authors:
+- Ingo Buerk:
+  name: "Ingo Buerk"
+- Daisy Tsang:
+  name: "Daisy Tsang"
+excerpt: 
+---
+
+{% toc %}
+
+# Introduction
+
+In [part one](/2021/09/06/connector-table-sql-api-part2) of this tutorial, you learned how to build a custom source connector for Flink. In part two, you will learn how to integrate the connector with a test email inbox through the IMAP protocol, filter out emails, and execute [Flink SQL on the Ververica Platform](https://www.ververica.com/apache-flink-sql-on-ververica-platform). 
+
+# Goals
+
+Part two of the tutorial will teach you how to: 
+
+- integrate a source connector which connects to a mailbox using the IMAP protocol
+- use [Jakarta Mail](https://eclipse-ee4j.github.io/mail/), a Java library that can send and receive email via the IMAP protocol  
+- write [Flink SQL](https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/table/sql/overview/) and execute the queries in the Ververica Platform
```

Comment:
explain why we use it here, mostly `in the Ververica Platform for nicer visualization`